gradle.properties

```diff
@@ -2,6 +2,16 @@ projectVersion=4.0.0-SNAPSHOT
 
 # This prevents the Grails Gradle Plugin from unnecessarily excluding slf4j-simple in the generated POMs
 # https://github.com/grails/grails-gradle-plugin/issues/222
+gradleToolingApiVersion=8.10.2
+grailsGradlePluginVersion=7.0.0-SNAPSHOT
+grailsVersion=7.0.0-SNAPSHOT
+groovyVersion=4.0.24
+javaxAnnotationApiVersion=3.0.0
```

Comment:
javax should not be used now; it looks like this is really the jakarta version.  These are included in the bom by default.